composer.json

```diff
@@ -19,5 +19,9 @@
     },
     "autoload-dev": {
         "psr-4": { "React\\Tests\\Dns\\": "tests" }
-    }
+    },
+    "config": {
+    "platform": {
+    "php":"5.3.30"
```

Comment:
Missing indention

Reply:
Thanks for spotting, I've removed the affected rows again (this was only a testing leftover) :shipit: